src/controller/home/homeController.ts

```diff
@@ -0,0 +1,56 @@
+import * as express from "express";
+import {Controller} from "./../controller";
+import {WeatherServiceInterface} from "./../../../src/library/services/weather/weatherServiceInterface";
+
+export class HomeController extends Controller {
+
+    private forecastService: WeatherServiceInterface;
+
+    constructor(forecastService: WeatherServiceInterface) {
+        super();
+        this.forecastService = forecastService;
+    }
+
+    public index(req: express.Request, res: express.Response) {
+
+        try {
+
+            res.render(
+                "default/index", {
+                    title: "ETaskr Test"
+                }
+            );
+
+        } catch (e) {
+            console.log(e);
+        }
+
+    }
+
+    public getTemperature(req: express.Request, res: express.Response) {
+
+        let lat = req.params.lat;
+        let long = req.params.long;
+
+        if (!lat || !long) {
+
+            res.json({
+                data: {},
+                status: -1,
+                statusMessage: "Didn't provide latitude or longitude",
+            });
```

Comment:
This is being treated as a proxy api therefore should have a standard JSON structure message for consumers.
{ data, status, statusMessage}